tests/integration/framework/process/placement/placement.go

```diff
@@ -164,3 +175,123 @@ func (p *Placement) InitialClusterPorts() []int {
 func (p *Placement) CurrentActorsAPILevel() int {
 	return 20 // Defined in pkg/actors/internal/api_level.go
 }
+
+func (p *Placement) EstablishConn(ctx context.Context) (*grpc.ClientConn, error) {
```

Comment:
Change `EstablishConn` to `GRPCClient` which returns a `placementv1pb.Client`. It seems like every consumer is using it for this anyway. If we really need a `*grpc.Conn` to placement we can keep it, and `GRPCClient` can consume is. If it is the case that `EstablishConn` is needed, consider renaming to `DialContext` to be consistent in naming.